external/java/src/main/java/ch/epfl/dedis/lib/crypto/Ed25519.java

```diff
@@ -25,6 +25,7 @@
     public static Field field = curve.getField();
     public static Point base = new Ed25519Point(Ed25519.ed25519.getB());
     public static Scalar prime_order = new Ed25519Scalar("EDD3F55C1A631258D69CF7A2DEF9DE1400000000000000000000000000000010", false);
+    public static Scalar cofactor = new Ed25519Scalar("0800000000000000000000000000000000000000000000000000000000000000", false);
```

Comment:
```go
// cofactor of the curve, as a ModInt
var cofactor = new(big.Int).SetInt64(8)
```

Are you sure about the zero ?

Reply:
yes, i encoded the scalar cofactor in Go and this is what I got

Reply:
oh man I just saw the 8 hidden in the zeros ...